e2e-cypress/Jenkinsfile.template

```diff
@@ -79,14 +95,18 @@ def stageTest(def context) {
       stash(name: "integration-test-reports-junit-xml-${context.componentId}-${context.buildNumber}", includes: 'build/test-results/integration-junit.xml', allowEmpty: true)
       stash(name: "acceptance-test-reports-junit-xml-${context.componentId}-${context.buildNumber}", includes: 'build/test-results/acceptance-junit.xml', allowEmpty: true)
 
+      sh 'npm run generate:pdf'
+      zip zipFile: 'cypress/pdf.zip', archive: false, dir: 'build/test-results/mochawesome/pdf'
+      archiveArtifacts artifacts: 'cypress/pdf.zip', fingerprint: true, daysToKeep: 2, numToKeep: 3
+
       if (fileExists('cypress/videos')) {
         zip zipFile: 'cypress/videos.zip', archive: false, dir: 'cypress/videos'
         stash(name: "acceptance-test-videos-${context.componentId}-${context.buildNumber}", includes: 'cypress/videos.zip', allowEmpty: true)
         archiveArtifacts artifacts: 'cypress/videos.zip', fingerprint: true, daysToKeep: 2, numToKeep: 3
       }
 
-      if (fileExists('cypress/screenshots')) {
-        zip zipFile: 'cypress/screenshots.zip', archive: false, dir: 'cypress/screenshots'
+      if (fileExists('build/test-results/screenshots')) {
+        zip zipFile: 'cypress/screenshots.zip', archive: false, dir: 'build/test-results/screenshots'
         stash(name: "acceptance-test-screenshots-${context.componentId}-${context.buildNumber}", includes: 'cypress/screenshots.zip', allowEmpty: true)
         archiveArtifacts artifacts: 'cypress/screenshots.zip', fingerprint: true, daysToKeep: 2, numToKeep: 3
```

Comment:
@roicarrera what's the intended purpose of mochawesome reports opposed to LeVA-based documentation? Why archive reports, videos, and screenshots in Jenkins where they keep consuming disk space, even only for two days, which could cause Jenkins to fail when the disk runs full?